src/shared/components/Post/Post.tsx

```diff
@@ -0,0 +1,75 @@
+import Image from "next/image";
+import classNames from "classnames/bind";
+import styles from "@/shared/components/Post/Post.module.scss";
+import getFullDate from "@/shared/utils/getDate";
+import Clock from "@/images/ic_clock.svg";
+import Location from "@/images/ic_location.svg";
+import NoticeMessage from "./NoticeMessage/NoticeMessage";
+import HighPriceRateBadge from "./HighPriceRateBadge/HighPriceRateBadge";
+
+const cn = classNames.bind(styles);
+
+interface PostProps {
+  imageUrl: string;
+  startsAt: string;
+  workhour: number;
+  hourlyPay: number;
+  closed: boolean;
+  name: string;
+  address1: string;
+  originalHourlyPay: number;
+}
+
+export default function Post({
+  imageUrl,
+  startsAt,
+  workhour,
+  hourlyPay,
+  closed,
+  name,
+  address1,
+  originalHourlyPay,
+}: PostProps) {
+  const currentDate = new Date();
+  const endDate = new Date(startsAt);
+  const isPast = currentDate > endDate;
+
+  return (
+    <div className={cn("postContainer", { closed, isPast })}>
+      <div className={cn("imgContainer")}>
+        <NoticeMessage isPast={isPast} closed={closed} />
+        <Image style={{ objectFit: "cover" }} src={imageUrl} alt="식당 공고" fill />
```

Comment:
일관되게 className으로 표현해주는게 좋을 것 같습니다~
```suggestion
        <Image style={{ objectFit: "cover" }} src={imageUrl} alt="식당 공고" fill />
```